src/constants.ts

```diff
@@ -17,6 +17,9 @@
 export const SOURCE = 'sourceChain';
 export const TARGET = 'targetChain';
 
+export const CHAIN_1 = '1';
+export const CHAIN_2 = '2';
+
```

Comment:
But it's all private usage. The only place where it is exported outside is via `chainNumber` which, honestly, given it's name I'd expect to be a `number`.

I don't get what problem it really solves - as mentioned in the other comment I think it would be better to define the expected type to be `'1'|'2'` instead of generic `string` & then define some constants to overcome an issue of "I don't know what `string` I need to pass to this function."

While I am a fan of constants for things like arbitrary strings or arbitrary numbers, the main purpose of that is to give them meaning, examples
```
const SEND_MESSAGE_RUNTIME_API = 'palletBridgeGrandpa::send_message';
const NUMBER_OF_SUPPORTED_CHAINS = 3;
```
In this example, we basically do:
```
export const ONE_CHARACTER_STRING_WITH_NUMBER_ONE = '1'
export const ONE_CHARACTER_STRING_WITH_NUMBER_TWO = '2'
```
for purpose of using it when calling private functions. That really sounds like too much. If you really think they do provide value, please at least move them to the `src/configs/index.ts` file and make them local. IMHO they should never be used again (if they are that indicates some more severe code issue) and they just pollute the "global" file.